charts/matrix-antiscam-bot/values.yaml

```diff
@@ -0,0 +1,7 @@
+environment: production
```

Comment:
I think it would be good to add defaults for all the values here. In this case helm lint wouldn't fail, but as a way of documenting how the chart is expected to work it can be helpful, wdyt?